packages/xo-web/src/common/intl/messages.js

```diff
@@ -1156,7 +1156,8 @@ const messages = {
   vmCoresPerSocket:
     '{nSockets, number} socket{nSockets, plural, one {} other {s}} with {nCores, number} core{nCores, plural, one {} other {s}} per socket',
   vmCoresPerSocketNone: 'None',
-  vmCoresPerSocketIncorrectValue: 'Incorrect cores per socket value',
+  vmCoresPerSocketIncorrectValue:
+    'nSockets * nCoresPerSocket should equal nVCpus',
```

Comment:
The translations will be invalid if you use the same key.

Reply:
You think that message is really clear for user?

Reply:
I think it can be improved. Can you please suggest an improvement?

Reply:
IMHO we should know the relationship between the number of cores per socket and maxVcpus because it's calculated from maxVcpus. 
The user doesn't know the number of cores per sockets or the number of sockets to solve this error.

Reply:
IMHO we should know the relationship between the number of cores per socket and nVcpus.  
The user doesn't know the number of cores per sockets or the number of sockets to solve this error.

Reply:
IMHO we should know the relationship between the number of cores per socket and nVcpus. 
The user doesn't know the number of cores per sockets or the number of sockets to solve this error.